operator_ui/src/pages/Jobs/RecentRuns.tsx

```diff
@@ -0,0 +1,151 @@
+import { CardTitle, KeyValueList } from '@chainlink/styleguide'
+import {
+  createStyles,
+  Theme,
+  Typography,
+  WithStyles,
+  withStyles,
+  Card,
+  Grid,
+} from '@material-ui/core'
+import Content from 'components/Content'
+import JobRunsList from 'components/JobRuns/List'
+import TaskList from 'components/Jobs/TaskList'
+import React from 'react'
+import { GWEI_PER_TOKEN } from 'utils/constants'
+import formatMinPayment from 'utils/formatWeiAsset'
+import { formatInitiators } from 'utils/jobSpecInitiators'
+import { JobData } from './sharedTypes'
+
+const totalLinkEarned = (job: NonNullable<JobData['jobSpec']>) => {
+  const zero = '0.000000'
+  const unformatted =
+    job.attributes.earnings &&
+    (job.attributes.earnings / GWEI_PER_TOKEN).toString()
+  const formatted =
+    unformatted &&
+    (unformatted.length >= 3 ? unformatted : (unformatted + '.').padEnd(8, '0'))
+  return formatted || zero
+}
+
+const chartCardStyles = (theme: Theme) =>
+  createStyles({
+    wrapper: {
+      marginLeft: theme.spacing.unit * 3,
+      marginTop: theme.spacing.unit * 2,
+      marginBottom: theme.spacing.unit * 2,
+    },
+    paymentText: {
+      color: theme.palette.secondary.main,
+      fontWeight: 450,
```

Comment:
Is our font family supports fontWeight: 450?

Reply:
I didn't change it, so I hope so 🙂